windows-wwiv-update/wwiv-update.cs

```diff
@@ -31,6 +32,8 @@ public partial class Form1 : Form
         public static string wwivUpdateVersion;
         public static string updateVersionLabel;
         public static string updateTagLabel;
+        //public static string appPath = Path.GetDirectoryName(Application.ExecutablePath);
```

Comment:
They are there for possible future enhancements to the code where I will be using them but not decided at this time no need for another global if it's not in use and might as well comment it out to show that it isn't in use so I don't forget it's there.